tests/expectations/metrics/test_metric_providers.py

```diff
@@ -0,0 +1,315 @@
+
+    registered_metric_keys = list(_registered_metrics.keys())
+    for key in registered_metric_keys:
+        assert "column_pair_values.equal_seven" not in key
+    prev_registered_metric_key_count = len(registered_metric_keys)
+
+    class CustomColumnPairValuesEqualSeven(ColumnPairMapMetricProvider):
+        condition_metric_name = "column_pair_values.equal_seven"
+
+        @column_pair_condition_partial(engine=PandasExecutionEngine)
+        def _pandas(cls, column_A, column_B, **kwargs):
+            raise NotImplementedError
+
+        @column_pair_condition_partial(engine=SqlAlchemyExecutionEngine)
+        def _sqlalchemy(cls, column_A, column_B, _dialect, **kwargs):
+            raise NotImplementedError
+
+        @column_pair_condition_partial(engine=SparkDFExecutionEngine)
+        def _spark(cls, column_A, column_B, **kwargs):
+            raise NotImplementedError
+
+    CustomColumnPairValuesEqualSeven()
+
+    assert len(_registered_metrics.keys()) == prev_registered_metric_key_count + 7
+
+    for key in _registered_metrics.keys():
+        if "column_pair_values.equal_seven" in key:
+            print(key)
+
+    new_keys = [
+        "column_pair_values.equal_seven.condition",
+        "column_pair_values.equal_seven.unexpected_count",
+        "column_pair_values.equal_seven.unexpected_index_list",
+        "column_pair_values.equal_seven.unexpected_index_query",
+        "column_pair_values.equal_seven.unexpected_rows",
+        "column_pair_values.equal_seven.unexpected_values",
+        "column_pair_values.equal_seven.filtered_row_count",
+    ]
+    for key in new_keys:
+        assert key in _registered_metrics.keys()
+
+
+def test__multicolumn_map_metric__registration():
+    """This tests whether the MultiColumnMapMetricProvider class registers the correct metrics."""
+
+    registered_metric_keys = list(_registered_metrics.keys())
+    for key in registered_metric_keys:
+        assert "multicolumn_values.equal_seven" not in key
+    prev_registered_metric_key_count = len(registered_metric_keys)
+
+    class CustomColumnPairValuesEqualSeven(MulticolumnMapMetricProvider):
+        condition_metric_name = "multicolumn_values.equal_seven"
+
+        condition_domain_keys = (
+            "batch_id",
+            "table",
+            "column_list",
+            "row_condition",
+            "condition_parser",
+            "ignore_row_if",
+        )
+        condition_value_keys = ()
+
+        @multicolumn_condition_partial(engine=PandasExecutionEngine)
+        def _pandas(cls, column_list, **kwargs):
+            raise NotImplementedError
+
+        @multicolumn_condition_partial(engine=SqlAlchemyExecutionEngine)
+        def _sqlalchemy(cls, column_list, **kwargs):
+            raise NotImplementedError
+
+        @multicolumn_condition_partial(engine=SparkDFExecutionEngine)
+        def _spark(cls, column_list, **kwargs):
+            raise NotImplementedError
+
+    CustomColumnPairValuesEqualSeven()
```

Comment:
@abegong Similarly, this was probably meant to be `CustomMulticolumnValuesEqualSeven()`.  Thank you.